### PR TITLE
Use pull request title for squash and merge commit message

### LINKED ISSUE #276: Force "squash and merge" title to be the pull request title
It does if there are multiple commits, but if there is only one commit, if will use the commit title instead. I often rename pull requests because submitters do a very bad job of naming their pull requests. When I then "squash and merge" it will instead choose the commit title, which is the original bad title, and I have to rename yet again. Would be nice if we could just copy in the pull request title when "Merge pull request" is clicked and you're in the confirm view.

### PR DESCRIPTION
**Pull request by hkdobrev**
Fixes #276

### CODE REVIEW

**Comment by hkdobrev** [comment]
@sindresorhus What do you think about this?

File: extension/content.js [hunk 1]
```diff
@@ -377,6 +377,12 @@ function removeProjectsTab() {
 	}
 }
 
+function fixSquashAndMergeTitle() {
+	$('.btn-group-squash button[type=submit]').click(() => {
+		$('#merge_title_field').val($('.js-issue-title').text().trim() + ' (' + $('.gh-header-number').text() + ')');
```
Comment (paulmolluzzo) [inline extension/content.js]: Any reason you chose not to use a template literal here?

```
$('#merge_title_field').val(`${$('.js-issue-title').text().trim()} (${$('.gh-header-number').text()})`);
```

Comment (hkdobrev) [inline extension/content.js]: Just the habit of too much writing on ES5. Will change that tomorrow.

**Comment by sindresorhus** [comment]
Can you add it to the feature highlight? No screenshot needed.

**Comment by hkdobrev** [comment]
@sindresorhus @paulmolluzzo Updated.

**Comment by sindresorhus** [comment]
Great! This is going to be sooo useful for me. I do this manually many times every day.

**Pull request merged**